src/components/CourseChecklist/CourseChecklist.scss

```diff
@@ -1,10 +1,16 @@
 @import 'edx-bootstrap';
+@import "~bootstrap/scss/_buttons";
+@import "~bootstrap/scss/utilities/_screenreaders";
 
-$dark-color: theme-color("dark");
+$light-color: theme-color("light");
 $lightest-color: theme-color("lightest");
 $primary-color: theme-color("primary");
 $success-color: theme-color("success");
 
+.checklist-item-link {
+  text-decoration: none;
```

Comment:
This removes the underline of the `a` element rendered by the `Hyperlink` element.